src/courses/advanced/09.md

```diff
@@ -31,17 +31,19 @@ Neither command has output, but you should see a new file if you examine your `.
 ```sh
 tree .github
 ```
+
 @tab Expected Output - .github folder structure
+
 ```
 .github
 └── workflows
     └── pipeline.yml
 ```
+
 :::
 
 Open that file up for editing.
 
-
 ### Workflow File - Complete Example
 
 For reference, this is the complete workflow file we will end up with at the end of the class:
```

Comment:
make sure that we call it `pipeline_answer_key.yml` now in the appropriate locations